ProjectSidewalk/SidewalkWebpage
# 2152 Mission screens

Resolves #2152

Implements the basic structure of mission screens to improve user education. 

##### Before/After screenshots (if applicable)
Before
<img width="1440" alt="Screen Shot 2022-11-16 at 11 11 08 PM" src="https://user-images.githubusercontent.com/8168506/202382657-02177a69-4413-4ac8-8996-2609dd0aaa5e.png">

After
<img width="1440" alt="Screen Shot 2022-11-16 at 11 25 33 PM" src="https://user-images.githubusercontent.com/8168506/202382689-d4b793ad-c9fa-4e4f-b2a8-ebb226d532bc.png">

##### Testing instructions
1. Test that all the mission screens for all the label types match figma designs.
2. Ensure that the text, emoji, and the on-image labels are correct in all the screens.
3. Ensure that the layout is optimum when the browser window is resized.

##### Things to check before submitting the PR
